x-pack/plugins/observability_solution/apm/public/components/alerting/ui_components/alert_details_app_section/latency_chart.tsx

```diff
@@ -142,7 +142,7 @@ function LatencyChart({
         <AlertActiveTimeRangeAnnotation
           alertStart={alert.start}
           alertEnd={alertEnd}
-          color={chroma(transparentize('#F04E981A', 0.2)).hex().toUpperCase()}
+          color={euiTheme.colors.danger}
```

Comment:
Opacity of this color is now handled in `x-pack/packages/observability/alert_details/src/components/alert_active_time_range_annotation.tsx`.

Before this change, annotation for Metric threshold had the following issue:
![image](https://github.com/elastic/kibana/assets/12370520/f902df47-b19d-474b-b644-aadd4f605465)

Reply:
When I was working on fixing recovered alert annotation in APM Latency chart, `euiTheme.colors.danger` with opacity 0.1 was not showing right color as I expected, so I used `chroma(transparentize('#F04E981A', 0.2)).hex().toUpperCase()` with opacity 1. But now it looks fine with `euiTheme.colors.danger` and opacity 0.1. Not sure what was the issue before. 🤔